lisk-dex-electron/src/SignInModal.jsx

```diff
@@ -191,22 +191,28 @@ export default class SignInModal extends React.Component {
     return (
       <>
         <div className="modal-background" />
-        <div id="sign-in-modal" className="modal-foreground">
-          <form onSubmit={this.handleSubmit}>
-            <div className="sign-in-progress-area">
+        <div className="modal-foreground">
+          <div id="sign-in-modal">
+            <div className="sign-in-wrapper">
               {this.state.signingIn && (
-                <div className="progress">
-                  <div className="indeterminate" />
+                <div className="sign-in-progress-area">
+                    <div className="progress">
+                      <div className="indeterminate" />
+                    </div>
                 </div>
               )}
+              <div className="close-btn">
```

Comment:
This is mainly because the `X` is a html entity and works with `font-size` it's height and width isn't equal. I changed to `padding` to the right a little in the css and it's looking better :)